src/lib/pages/backoffice/Loan/LoanDetails/LoanActions/LoanActions.js

```diff
@@ -1,14 +1,91 @@
 import { OverdueLoanSendNotificationModal } from '@modules/Loan/backoffice/OverdueLoanSendNotificationModal';
 import { InfoMessage } from '@components/backoffice/InfoMessage';
+import { withCancel } from '@api/utils';
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import { documentApi } from '@api/documents';
 import { List, Button } from 'semantic-ui-react';
 import { omit } from 'lodash/object';
 import { CancelModal } from '@components/CancelModal';
+import { ConfirmModal } from '@components/ConfirmModal';
 import _isEmpty from 'lodash/isEmpty';
 import capitalize from 'lodash/capitalize';
 
 export default class LoanActions extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      overbooked: false,
+    };
+  }
+
+  componentDidMount() {
+    this.fetchCirculation();
+  }
+
+  componentWillUnmount() {
+    this.cancellableCirculation && this.cancellableCirculation.cancel();
+  }
+
+  fetchCirculation = async () => {
+    const { loanDetails } = this.props;
+    const { document_pid } = loanDetails.metadata;
+    try {
+      this.cancellableCirculation = withCancel(documentApi.get(document_pid));
+      const response = await this.cancellableCirculation.promise;
+      const circulation = response.data.metadata.circulation;
+      this.setState({ overbooked: circulation.overbooked === true });
+    } catch (error) {
+      if (error !== 'UNMOUNTED') {
+        this.setState({ overbooked: false });
```

Comment:
so if we have an error then a librarian will be able to extend a loan without knowing the document is overbooked?

Reply:
I don't think the user would be able to extend the loan if that code was executed. The catch was more intended for consistency with the rest of the code.